log-base/src/Log/Class.hs

```diff
@@ -39,6 +39,8 @@ class MonadTime m => MonadLog m where
   localData   :: [Pair] -> m a -> m a
   -- | Extend the current application domain locally.
   localDomain :: T.Text -> m a -> m a
+  -- | Capture current context of a logger for use in the 'IO' monad.
```

Comment:
I'd reword this comment as "Returns an IO action that logs messages using the current `MonadLog` context. Useful for interfacing with libraries such as `aws` or `amazonka` that accept logging callbacks that operate in IO." to make the purpose of this function more clear.